### ISSUE TITLE
There's two quests for getting witchery familiars

### ISSUE THREAD

**Issue by Prometheus0000** [GTNewHorizons/NewHorizons#5576]
#### Which modpack version are you using?
2.0.8.4
#
I didn't notice the existing one and added a second one as part of the biome changing questline. Sorry. #5341 
#
#### What do you suggest instead/what changes do you propose?
Get rid of one, merge them, whatever. Probably get rid of the old one.

**Comment by Dream-Master** [comment]
@Prometheus0000 you know now how to make quests. Go and fix it please and make a pull request. Thanks

**Comment by Prometheus0000** [comment]
@Dream-Master @richardhendricks 
I originally talked to moron about it since I wasn't sure whether it was ok for me to get rid of/hide old quests. Anyways I followed the instructions to remove prereqs for the old quest and set it to hidden, but is there a way to remove it from the tab completely?

I used remove quest, and it removed it from the page, and I confirmed it's still in the database like it should be, so it won't mess up the numbering. Was this step just missing from the instructions?
